module-2-html/exercises/03-recipe/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Apple pie recipe</title>
+</head>
+<body>
+    <h1>Apple pie</h1>
+    <br>
+    <img src="apple-pie.jpg" alt="Apple pie" style="max-width: 100vw;">
+    <br><br>
+    <h3>Total Time</h3>
+    <p>Prep: 20 min. Bake: 45 min.</p>
+    <br>
+    <h3>Makes</h3>
+    <p>8 servings</p><br>
+    <h3>Ingredients</h3>
+    <ul>
+        <li>1/2 cup <em>sugar</em></li>
+        <li>1/2 cup <em>packed brown sugar</em></li>
+        <li>3 tablespoons <em>all-purpose flour</em></li>
+        <li>1 teaspoon <em>ground cinnamon</em></li>
+        <li>1/4 teaspoon <em>ground ginger</em></li>
+        <li>1/4 teaspoon <em>ground nutmeg</em></li>
+        <li>6 to 7 cups <em>thinly sliced peeled tart apples</em></li>
+        <li>1 tablespoon <em>lemon juice</em></li>
+        <li>Pastry for <em>double-crust pie (9 inches)</em></li>
+        <li>1 tablespoon <em>butter</em></li>
+        <li>1 large <em>egg white</em></li>
+        <li><em>Additional sugar</em></li>        
+    </ul>
+    <br>
+    
+    <h3>Directions</h3>
+    <p>In a <em>small bowl</em>, combine the sugars, flour and spices; set aside. In a <em>large bowl</em>, toss apples with lemon juice. Add <em>sugar mixture</em>; toss to coat.</p>
+    <p>Line a 9-in. pie plate with <em>bottom crust</em>; trim even with edge. Fill with <em>apple mixture</em>; dot with butter. Roll remaining crust to fit top of pie; place over filling. Trim, seal and flute edges. <em>Cut slits in crust</em>.</p>
+    <p>Beat <em>egg white</em> until foamy; brush over crust. Sprinkle with sugar. <em>Cover edges loosely</em> with foil.</p>
+    <p>Bake at 375° for 25 minutes. Remove foil and bake until crust is <em>golden brown</em> and filling is bubbly, 20-25 minutes longer. Cool on a <em>wire rack</em>.</p>

```

Comment:
Predložio bih da se upute označe kao brojčana lista (`<ol>`) zato što je redoslijed koraka bitan za ispravan prijenos informacije u receptu.